ultravox/model/ultravox_config.py

```diff
@@ -99,7 +99,6 @@ def __init__(
         audio_model_id: Optional[str] = None,
         text_model_id: Optional[str] = None,
         ignore_index: int = -100,
-        audio_token_index: int = 32000,
```

Comment:
We have not been using `audio_token_index` for a while now.

Reply:
Also, as the tests also verify, removing this will not cause issues when `audio_token_index` is actually present in `config.json`.

Reply:
As the Mistral experimentation uncovered, `audio_token_index` or something like it would be very useful downstream to have a token ID that is sensible to use as a placeholder with the particular tokenizer. In vLLM we currently hardcode the "<|reserved_special_token_0|>" Llama token, which broke when I tried to use a Mistral LLM.

Reply:
That said, we don't currently use this in vLLM, so removing it won't break anything to my knowledge.

Reply:
Okay good to know. I'll keep it then.

Reply:
Actually I'll remove it for now and then we'll add it back in coordination with vllm, because the current value (32000) is probably just plain wrong.